README.md

```diff
@@ -74,6 +75,16 @@ overwrite it with the environment variable like:
 DD_API_KEY=12345678990 ./bin/agent/agent run -c bin/agent/dist/datadog.yaml
 ```
 
+If the Datadog Agent is not installed, or if you do not want to edit your `datadog.yaml`, 
+a configuration file is available at `bin/agent/dist/datadog.yaml`. To run the Agent with this configuration file,
```

Comment:
I realize now that this is not entirely correct either.

Basically we should recommend creating a development `datadog.yaml` in `dev/dist/`, which should contain at least a valid API key. It will then be copied to `bin/agent/dist/` by the `invoke agent.build` command, and there won't be any need to provide `DD_API_KEY` on the command line.

So if we want the full step-by-step instructions, we should recommend:
1. create a development `datadog.yaml` in `dev/dist/`. It should contain at least a valid API key, i.e.:

```yaml
api_key: <API_KEY>
``` 

2. build the agent with `invoke agent.build`. This will also copy the development `dev/dist/datadog.yaml` file to `bin/agent/dist/`.
3. start the dev agent with `bin/agent/agent run -c bin/agent/dist/`. While the dev agent is running, you can run commands against it with `bin/agent/agent <command> -c bin/agent/dist/`.